### DISCUSSION TITLE
Important Update on rnmapbox Maintenance and Contribution Opportunities

### DISCUSSION THREAD

**Discussion by mfazekas** [rnmapbox/maps#3645] [Announcements]
Dear rnmapbox Community,

I hope this message finds you well. I wanted to share some important updates regarding my availability and the future of the rnmapbox project.

As many of you know, I have been balancing multiple responsibilities, including a half-time gig, family commitments, and my passion for badminton coaching. Despite my best efforts - which could have been better focused on marketing rather than development - maintaining rnmapbox has not been financially viable. Consequently, I have accepted a full-time position unrelated to rnmapbox, which significantly limits the time I can dedicate to this project.

Moving forward, I will focus on the following areas:

- **Helping Contributors:** Ensuring continuous integration (CI) is running smoothly,  and reviewing/helping pull requests (PRs).
- **Prioritizing Issues/Questions in Sponsors Repo:** Sponsors contributing over $100 already have access to this repository and will continue to receive prioritized support.

I want to extend my heartfelt thanks to all contributors and sponsors. Your support and contributions have been invaluable in keeping the project alive and thriving. If you have been considering contributing to rnmapbox, now is a great time to get involved. Your contributions are invaluable and can make a significant impact on the project’s success.

Thank you for your continued support and understanding.

**Comment by Gnative** [comment]
@mfazekas thank you for all the hard work and what you have done for the react native community.

**Comment by JamesIde** [comment]
Thank you @mfazekas for all your hard work over the years. Echoing what @Gnative said - your work has been invaluable for the React Native community and us niche Mapbox folk.

**Reply by JamesIde** [reply]
It may also be worth announcing this in the Mapbox discord so more people are across it, as the RN channel does gets posts and questions almost daily. I'm sure the people at Mapbox (Chris or Dan from the discord) would appreciate knowing and perhaps give this area a bit more exposure.
